packages/muelu/src/Misc/MueLu_InverseApproximationFactory_def.hpp

```diff
@@ -93,7 +97,7 @@ namespace MueLu {
 
     // check which approximation type to use
     const std::string method = pL.get<std::string>("inverse: approximation type");
-    TEUCHOS_TEST_FOR_EXCEPTION(method != "diagonal" && method != "lumping", Exceptions::RuntimeError,
+    TEUCHOS_TEST_FOR_EXCEPTION(method != "diagonal" && method != "lumping" && method != "spai", Exceptions::RuntimeError,
```

Comment:
Maybe spell out the "spai" identifier as "sparse approximate inverse". It's longer, but also much more descriptive.